viral/learning_stages.py

```
@@ -22,112 +31,173 @@
+            arr=data_matrix,
+            x_axis=np.linspace(config.start, config.end, data_matrix.shape[1]),
+            color=colors[color_idx],
+            label=stage_name,
+        )
+        color_idx += 1
+        for landmark_center in [45, 90, 135]:
+            plt.axvspan(
+                landmark_center - 2.5, landmark_center + 2.5, color="red", alpha=0.5
+            )
+
+    1 / 0
```

Ruff (0.14.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)